### PR TITLE
New Adapter: Driftpixel

### PR DESCRIPTION
**Pull request by driftpixelai**
Dev docs: https://github.com/prebid/prebid.github.io/pull/5302

### CODE REVIEW

File: static/bidder-info/driftpixel.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+endpoint: "http://rtb.driftpixel.live?pid={{.SourceId}}&host={{.Host}}&pbs=1"
+maintainer:
+  email: "developer@driftpixel.ai"
```
Comment (Sonali-More-Xandr) [inline static/bidder-info/driftpixel.yaml:3]: @driftpixelai We have sent an email to `developer@driftpixel.ai` for verification. Please reply to the mail with 'Received' message to complete the verification process.

Thank you!

File: static/bidder-info/driftpixel.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+endpoint: "http://rtb.driftpixel.live?pid={{.SourceId}}&host={{.Host}}&pbs=1"
```
Comment (onkarvhanumante) [inline static/bidder-info/driftpixel.yaml:1]: endpoint is reachable 

```
curl -i --location --request POST 'http://rtb.driftpixel.live?pid=1&host=foo&pbs=1'
HTTP/1.1 400 Bad Request
Server: nginx
Date: Thu, 23 May 2024 10:26:53 GMT
Transfer-Encoding: chunked
Connection: keep-alive
```

**Comment by onkarvhanumante** [comment]
@driftpixelai should update json test
- to have request with multiple imps
- to have multi-format imp request

File: static/bidder-info/driftpixel.yaml (hunk 1 above)
Comment (Sonali-More-Xandr) [reply static/bidder-info/driftpixel.yaml:3]: @driftpixelai  Please respond to this mail to complete the verification process.

**Review by onkarvhanumante — APPROVED** [review]

File: static/bidder-info/driftpixel.yaml (hunk 1 above)
Comment (dimashirokov) [reply static/bidder-info/driftpixel.yaml:3]: Hi @Sonali-More-Xandr, have you received the response?

**Review by Sonali-More-Xandr — APPROVED** [review]

**Pull request merged**